README.md

```diff
@@ -57,81 +50,34 @@ Here's how to build the StableHLO repo on Linux or macOS:
 
    Cloning the LLVM repository may take a few minutes.
 
-4. Make sure you check out the correct commit in the LLVM repository:
+3. Make sure you check out the correct commit in the LLVM repository:
 
    ```sh
    (cd llvm-project && git fetch && git checkout $(cat ../build_tools/llvm_version.txt))
    ```
 
    You need to do this every time `llvm_version.txt` changes.
 
-5. Configure and build MLIR:
-
-   ```sh
-   MLIR_ENABLE_BINDINGS_PYTHON=OFF build_tools/build_mlir.sh ${PWD}/llvm-project/ ${PWD}/llvm-build
-   ```
-
-   This will take a considerable amount of time. For example, on a MacBook Pro
-   with an M1 Pro chip, building MLIR took around 10 minutes at the moment
-   of writing.
-
-   Again, you need to do this every time `llvm_version.txt` changes.
-
-6. Build StableHLO as a standalone library:
+4. Build StableHLO as a standalone library and run all the tests:
 
    ```sh
-   mkdir -p build && cd build
-
-   cmake .. -GNinja \
-     -DLLVM_ENABLE_LLD="$LLVM_ENABLE_LLD" \
-     -DCMAKE_BUILD_TYPE=Release \
-     -DLLVM_ENABLE_ASSERTIONS=ON \
-     -DSTABLEHLO_ENABLE_BINDINGS_PYTHON=OFF \
-     -DMLIR_DIR=${PWD}/../llvm-build/lib/cmake/mlir
-
-   cmake --build .
-   ```
-
-   If you are actively developing StableHLO, you may want the following additional
-   CMake settings:
-
-   ```sh
-   cmake .. -GNinja \
-     -DSTABLEHLO_ENABLE_LLD=ON \
-     -DCMAKE_BUILD_TYPE=RelWithDebInfo \
-     -DLLVM_ENABLE_ASSERTIONS=ON \
-     -DSTABLEHLO_ENABLE_BINDINGS_PYTHON=OFF \
-     -DSTABLEHLO_ENABLE_SPLIT_DWARF=ON \
-     -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
-     -DCMAKE_C_COMPILER_LAUNCHER=ccache \
-     -DSTABLEHLO_ENABLE_SANITIZER=address \
-     -DMLIR_DIR=${PWD}/../llvm-build/lib/cmake/mlir
-
-   cmake --build .
-   ```
-
-      This will enable debug symbols and ccache, which can speed up incremental
-      builds. It also creates a GDB index file in the binary to speed up
-      debugging.
-
-      If you build MLIR using the script above it should also set by default
-      `LLVM_USE_SPLIT_DWARF` which does the majority of the size saving for
-      the binary and should also be set.
-
-7. Now you can make sure it works by running some tests:
-
-   ```sh
-   ninja check-stablehlo-tests
+   # first configure the build system
+   cmake --preset debug
+   # then build the project
+   cmake --build ./build --target check-stablehlo-ci
    ```
 
    You should see results like this:
 
    ```txt
-   Testing Time: 5.99s
-     Passed: 47
+   Testing Time: 4.13s
+
+   Total Discovered Tests: 137
+   Passed: 137 (100.00%)
```

Comment:
Nit: This would be accurate if we were running `check-stablehlo-tests`, but because we are running `check-stablehlo-ci` 4 test suites need to run.

IMO, `check-stablehlo-tests` is more appropriate because the other 3 suites (tosa, linalg, testdata) are not as relevant to day-to-day development.

Also I commented about this elsewhere, and IDK if it's because we're using a `debug` build or something else, but running these tests is usually way faster for me. On `main`, I can run all 3888 tests in under 10 seconds, but with this branch it takes over a minute.

Reply:
Let's add a release preset and test against that?
I think starting the debug binaries are expensive.